### PR TITLE
DOC: ES01 fix for HDFS Methods

### PR DESCRIPTION
**Pull request by YashpalAhlawat**
All ES01 Errors resolved in the following cases:

scripts/validate_docstrings.py --format=actions --errors=ES01  pandas.HDFStore.put
scripts/validate_docstrings.py --format=actions --errors=ES01  pandas.HDFStore.get
scripts/validate_docstrings.py --format=actions --errors=ES01  pandas.HDFStore.info
scripts/validate_docstrings.py --format=actions --errors=ES01  pandas.HDFStore.keys

- [x] xref #57440 
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/io/pytables.py [hunk 1]
```diff
@@ -1647,6 +1660,10 @@ def info(self) -> str:
         """
         Print detailed information on the store.
 
+        This method provide convenient ways to interact with data
```
Comment (bergnerjonas) [inline pandas/io/pytables.py:1663]:
```suggestion
        This method provides convenient ways to interact with data
```

File: pandas/io/pytables.py [hunk 2]
```diff
@@ -1647,6 +1660,10 @@ def info(self) -> str:
...
+        stored in HDFStore files,allowing users to easily access
```
Comment (bergnerjonas) [inline pandas/io/pytables.py:1664]:
```suggestion
        stored in HDFStore files, allowing users to easily access
```

File: pandas/io/pytables.py [hunk 3]
```diff
@@ -640,6 +640,12 @@ def keys(self, include: str = "pandas") -> list[str]:
         """
         Return a list of keys corresponding to objects stored in HDFStore.
 
+        The include parameter can be used to filter the returned
```
Comment (bergnerjonas) [inline pandas/io/pytables.py:643]:
```suggestion
        The ``include`` parameter can be used to filter the returned
```

File: pandas/io/pytables.py [hunk 4]
```diff
@@ -640,6 +640,12 @@ def keys(self, include: str = "pandas") -> list[str]:
...
+        list based on the type of objects to include. If include
```
Comment (bergnerjonas) [inline pandas/io/pytables.py:644]:
```suggestion
        list based on the type of objects to include. If ``include``
```

File: pandas/io/pytables.py [hunk 5]
```diff
@@ -640,6 +640,12 @@ def keys(self, include: str = "pandas") -> list[str]:
...
+        is set to "pandas" (default), only pandas objects will
+        be included in the list. If include is set to "native",
```
Comment (bergnerjonas) [inline pandas/io/pytables.py:646]:
```suggestion
        be included in the list. If ``include`` is set to "native",
```

**Review by bergnerjonas — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks for the effort here, but it was decided that ES01 is not going to be enforced so closing. Happy to have PRs on other doc issues!

**Pull request closed without merging**